### PR TITLE
Initial version of HurtData.cs, FlickData.cs and SpeedData.cs. Review…

### PR DESCRIPTION
**Pull request by Daniel-Brum**
… pending.

@ninito-ph, this is how I'm supposed to make a pull request, right?

Thanks in advance,
Daniel Brum

### CODE REVIEW

File: Assets/Scripts/ScriptableObjects/Data/FlickData.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(filename = "Flick Data", menuname = "Flick Data", order = 0)]
```
Comment (ninito-ph) [inline Assets/Scripts/ScriptableObjects/Data/FlickData.cs]: Prefix menuname by the folder structure "Entity/...", like in the EffectData, ItemData or StatusEffectData scripts.

File: Assets/Scripts/ScriptableObjects/Data/HurtData.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(filename = "Hurt Data", menuname = "Hurt Data", order = 0)]
```
Comment (ninito-ph) [inline Assets/Scripts/ScriptableObjects/Data/HurtData.cs]: Prefix menuname by the folder structure "Entity/...", like in the EffectData, ItemData or StatusEffectData scripts.

File: Assets/Scripts/ScriptableObjects/Data/SpeedData.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(filename = "Speed Data", menuname = "Speed Data"), order = 0]
```
Comment (ninito-ph) [inline Assets/Scripts/ScriptableObjects/Data/SpeedData.cs]: Prefix menuname by the folder structure "Entity/...", like in the EffectData, ItemData or StatusEffectData scripts.

**Review by ninito-ph — CHANGES_REQUESTED** [review]
Good code, needs minor fixing.

File: Assets/Scripts/ScriptableObjects/Data/FlickData.cs (hunk 1 above)
Comment (Daniel-Brum) [reply Assets/Scripts/ScriptableObjects/Data/FlickData.cs]: Done. Thanks!

File: Assets/Scripts/ScriptableObjects/Data/HurtData.cs (hunk 2 above)
Comment (Daniel-Brum) [reply Assets/Scripts/ScriptableObjects/Data/HurtData.cs]: Done. Thanks!

File: Assets/Scripts/ScriptableObjects/Data/SpeedData.cs (hunk 3 above)
Comment (Daniel-Brum) [reply Assets/Scripts/ScriptableObjects/Data/SpeedData.cs]: Done. Thanks!

**Review by ninito-ph — APPROVED** [review]
Approved.

**Comment by Daniel-Brum** [comment]
The last commit solves all pending changes. Thanks a lot for reviewing, Paulo.

**Pull request merged**